app-office/fcrm/additional-files/fcrm.rdef.in

```diff
@@ -0,0 +1,40 @@
+resource app_signature "application/x-vnd.F-crm";
+
+resource app_flags B_SINGLE_LAUNCH;
+
+resource app_version {
+	major  = 0,
+	middle = 2,
+	minor  = 0,
+
+	/* 0 = development	1 = alpha			2 = beta
+	   3 = gamma		4 = golden master	5 = final */
+	variety = 5,

```

Comment:
I think variety should be 2 here (since it's a beta release) :)

Reply:
ok done.